packages/lumx-react/src/components/select/Select.tsx

```diff
@@ -238,233 +49,128 @@ const stopPropagation = (evt: Event) => evt.stopPropagation();
-        isValid,
-        hasError,
-        onClear,
-        onInputClick,
-        theme,
-        placeholder,
-        handleKeyboardNav,
-        targetUuid,
-        anchorRef,
-    ]);
 
-    return (
-        <div
-            ref={selectRef}
-            className={classNames(
-                className,
-                handleBasicClasses({
-                    hasError,
-                    hasInputClear,
-                    hasLabel: Boolean(label),
-                    hasMultiple: !isEmpty && isMultiple,
-                    hasPlaceholder: Boolean(placeholder),
-                    hasUnique: !isEmpty && !isMultiple,
-                    hasValue: !isEmpty,
-                    isDisabled,
-                    isEmpty,
-                    isFocus,
-                    isOpen,
-                    isValid,
-                    prefix: CLASSNAME,
-                    theme: theme === Theme.light ? Theme.light : Theme.dark,
-                }),
-                { [`${CSS_PREFIX}-custom-colors`]: useCustomColors },
-            )}
-            {...props}
-        >
-            {createParentElement()}
-            <Dropdown
-                closeOnClick={true}
-                closeOnEscape={true}
-                placement={Placement.AUTO_START}
-                showDropdown={isOpen!}
-                anchorRef={anchorRef}
-                onClose={onClose}
-                onInfiniteScroll={onInfiniteScroll}
-            >
-                {children}
-            </Dropdown>
-            {hasError && error && (
-                <InputHelper className={`${CLASSNAME}__helper`} kind={Kind.error} theme={theme}>
-                    {error}
-                </InputHelper>
+                        <div className={`${CLASSNAME}__input-indicator`}>
+                            <Icon icon={mdiMenuDown} size={Size.s} />
+                        </div>
+                    </div>
+                </>
             )}
-            {helper && (
-                <InputHelper className={`${CLASSNAME}__helper`} theme={theme}>
-                    {helper}
-                </InputHelper>
+
+            {variant === SelectVariant.chip && (
+                <Chip
+                    id={targetUuid}
+                    isSelected={!isEmpty}
+                    after={<Icon icon={isEmpty ? mdiMenuDown : mdiCloseCircle} />}
+                    onAfterClick={isEmpty ? onInputClick : onClear}
+                    onClick={onInputClick}
+                    chipRef={anchorRef as RefObject<HTMLAnchorElement>}
+                    theme={theme}
+                >
+                    {isEmpty && <span>{label}</span>}
+
+                    {!isEmpty && <span>{selectedValueRender!(value)}</span>}
+                </Chip>
             )}
-        </div>
+        </>
     );
 };
-Select.displayName = COMPONENT_NAME;
 
-export { CLASSNAME, DEFAULT_PROPS, Select, SelectProps, SelectVariant };
+const SelectWithContext = (props: SelectProps) => {
```

Comment:
I'm not a fan of this name. Could this component be just `Select`? the other component could be `SelectField` or `SelectInput`
 (It also goes for the SelectMultiple)

Reply:
Yes totally agree!
I changed the name for 'SelectField'